wordpress-mobile/WordPress-Android
# Reader move tab layout to parent part 3

Partially fixes #11849

This is a third PR in a series of PRs which is trying to introduce a regular TabLayout+ViewPager2 in the main Reader view. The current implementation uses FilteredRecyclerView component (which is not a RecyclerView :P) that has some downsides (eg. swipe gesture doesn't work).

This PR moves NewsCard from the ReaderPostListViewModel to NewsCardViewModel.

To test:
Prerequisities
- Open LocalNewsItem.kt and increment the `version` field
- Build the app locally

1. Open the app on the Reader tab
2. Make sure the NewsCard is shown only on the firstly selected tab
3. Restart the App and make sure the card is still being shown
4. Click on the "Read More" button and verify it opens a webbrowser
5. Dismiss the card and make sure it disappears (it doesn't re-appear after rotation or restart)

PR submission checklist:
